pysumma/ensemble.py

```diff
@@ -218,16 +218,26 @@ def attribute_product(list_config):
             {'attributes': d} for d in product_dict(**list_config)}
 
 
-def total_product(dec_conf={}, param_conf={}, attr_conf={}):
+def file_manager_product(list_config):
```

Comment:
Can you explain the format of list configuration for file_manager_product?

Reply:
I added a docstring to address this

Reply:
Looks great. You can close this pull request.